content/post/activity_hours.md

```diff
@@ -57,24 +45,25 @@ It normally takes less than 5 minutes to get the cluster running and fully opera
 
 The next step is to download the datasets. data.world has a size limit which prevents from uploading files bigger than 50MB, so we decided to upload ours to Google Drive for ease of use.
 
-<pre class="verbatim">
+```
```

Comment:
Not so simple... See the `font-size: 0.85em;`? Otherwise, it explodes. Need to see how it looks like in the widened layout